packages/control-input/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 ## Versions
 
+* [v3.0.0 - Use explicit labelling instead of implicit labelling for control inputs](#v300)
```

Comment:
Is this really a major change?

Reply:
Yes it is because we are re-ordering our markup